Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -188,7 +188,7 @@ public void ComeToStop()
 
 	public void ClimbCheck()
 	{
-		if (isGrabbing && ( ( actor as PlayerActor ) == null || ( actor as PlayerActor ).actorStats.CanUseStamina() ) )
+		if (isGrabbing && ( ( actor as PlayerActor ) == null || ( actor as PlayerActor ).actorStats.CanUseStat( Stat.Stamina ) ) )
```

Comment:
Yeah, this is something that I'm taking care of in the physic refactoring. Checking stats shouldn't even be happening within `ActorPhysics`, it should take place in `PlayerActorPhysics`.